_gsocblogs/2022/blog_Geant4_DivyanshTiwari.md

```diff
@@ -0,0 +1,38 @@
+---
+project: Geant4
+title: Symplectic Integrators
+author: Divyansh Tiwari
+date: 27.07.2022
+year: 2022
+layout: blog_post
+logo: Geant4-logo.png
+intro: |
+  Implementing Symplectic Integrators 
```

Comment:
The introduction must not be just a title, it should contain a sentence telling the readers what they are going to read about.

Reply:
Made the requested change